content/docs/hooks-reference.md

```diff
@@ -194,7 +194,7 @@ function reducer(state, action) {
 }
 
 function Counter({initialCount}) {
-  const [state, dispatch] = useReducer(reducer, initialState);
+  const [state, dispatch] = useReducer(reducer, {count: initialCount});
```

Comment:
This doesn't feel right though. 
In this example `initialState` is only used for reducer's `reset` action. Which will reset to an incorrect state as the `initialCount` may be different. IMHO in this example `Counter` should have *no props* as the correct example for `initialCount` is the one below with the `initialAction`